tests/functional/plugins/motherduck/test_motherduck_attach.py

```diff
@@ -93,17 +93,17 @@ def run_dbt_scope(self, project, database_name):
         # CREATE DATABASE does not work with SaaS mode on duckdb 1.0.0
         # This will be fixed in duckdb 1.1.0
```

Comment:
Really 1.1.1?

Reply:
This actually doesn't need to go here so I removed it!